src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -72,8 +84,36 @@ public PersonCard(Person person, int displayedIndex) {
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
 
         // Add module role pairs
+        Integer roleLabelFontSize = 14;
         person.getModuleRoleMap().getData().stream()
                 .sorted(Comparator.comparing(moduleRolePair -> moduleRolePair.moduleCode.toString()))
-                .forEach(moduleRolePair -> roles.getChildren().add(new Label(moduleRolePair.toString())));
+                .forEach(moduleRolePair -> {
+                    Label curLabel = new Label(moduleRolePair.toString());
+
+                    curLabel.setStyle(curLabel.getStyle() + " -fx-font-size: " + roleLabelFontSize.toString());
```

Comment:
Setting font size here, so that I can define a variable `roleLabelFontSize` and enforce that the same value is set for the font size and the icon height